modules/geo-layers/src/tile-layer/utils.js

```diff
@@ -125,7 +125,7 @@ function getIdentityTileIndices(viewport, z, tileSize, extent) {
  * return tiles that are on maxZoom.
  */
 export function getTileIndices({viewport, maxZoom, minZoom, zRange, extent, tileSize = TILE_SIZE}) {
-  let z = Math.ceil(viewport.zoom);
+  let z = Math.floor(viewport.zoom + Math.log2(TILE_SIZE / tileSize));
```

Comment:
Why `floor` instead of `round`? For tile size 512, this produces a 1x overzoom instead of a .5x overzoom (using the language that @kylebarron used).

Reply:
(The `image-tile` demo is 512)

Reply:
I thought @Pessimistress had suggested 
```
Math.round(viewport.zoom + Math.log2(512 / tileSize))
```

Reply:
Changed to `Math.round`... I need to remove `tileSize: 256` from the map tile demo because it's overloading.

Reply:
What do you mean by overloading?

Reply:
Additionally, a good example of being able to now pass an array to the `data` prop... You can change the map-tile data prop from
https://github.com/visgl/deck.gl/blob/19be47d9a700e64ba8384d9b787dc05c2b549175/examples/website/map-tile/app.js#L49
to

```js
data: [
  'https://a.tile.openstreetmap.org/{z}/{x}/{y}.png', 
  'https://b.tile.openstreetmap.org/{z}/{x}/{y}.png', 
  'https://c.tile.openstreetmap.org/{z}/{x}/{y}.png']
```

That might resolve part of your overloading issue.